### PR TITLE
Expose validator metrics

### PR DESCRIPTION
**Pull request by ri-ch**

### CODE REVIEW

File: coordinator/src/routes/health/get-status.ts [hunk 1]
```diff
@@ -30,5 +30,25 @@ export default [
         commitId: process.env.COMMIT_ID
       })
     }
+  },
+  {
+    method: "GET",
+    path: "/_validatormetrics/{path*}",
```
Comment (dan-jones-iw) [inline coordinator/src/routes/health/get-status.ts:36]: `/_validatormetrics` seems a bit ugly, can we capitalise M maybe? or use a dash like in `/$process-message`?

File: proxies/live/apiproxy/targets/apim.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
           <Name>RaiseFault.403Forbidden</Name>
         </Step>
       </Request>
-      <Condition>(proxy.pathsuffix MatchesPath "/_status") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
+      <Condition>(proxy.pathsuffix MatchesPath "/_status") and (proxy.pathsuffix MatchesPath "/_validatormetrics") and ((request.verb = "GET") or (request.verb = "HEAD"))</Condition>
```
Comment (dan-jones-iw) [inline proxies/live/apiproxy/targets/apim.xml:13]: how can `proxy.pathsuffix` match both `/_status` and `/_validatormetrics`

File: coordinator/src/routes/health/get-status.ts (hunk 1 above)
Comment (dan-jones-iw) [reply coordinator/src/routes/health/get-status.ts:36]: its not super clear what kind of values `path` would have? are there multiple endpoints on the validator that report metrics?

**Review by dan-jones-iw — DISMISSED** [review]

File: proxies/live/apiproxy/targets/apim.xml (hunk 2 above)
Comment (ri-ch) [reply proxies/live/apiproxy/targets/apim.xml]: Good spot. It cant. That should be an 'or' in between.

File: coordinator/src/routes/health/get-status.ts (hunk 1 above)
Comment (ri-ch) [reply coordinator/src/routes/health/get-status.ts:36]: It is a bit ugly. I dont think the name is too important, this is an internal api for diagnostics, and we might not even go to production with it

Comment (ri-ch) [reply coordinator/src/routes/health/get-status.ts:36]: `{path*}` just picks up any path components and proxies them onto the metrics endpoint

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**